File: src/main/java/no/nav/melosys/eessi/models/sed/nav/AapenPeriode.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonInclude(Include.NON_NULL)
public class AapenPeriode {

    @JsonProperty("type")
    private String ukjentEllerÅpenSluttdato;

    private String startdato;

    public String getUkjentEllerÅpenSluttdato() {
        return ukjentEllerÅpenSluttdato;
    }

    public void setUkjentEllerÅpenSluttdato(String ukjentEllerÅpenSluttdato) {
        this.ukjentEllerÅpenSluttdato = ukjentEllerÅpenSluttdato;
    }

    public String getStartdato() {
        return startdato;
    }

    public void setStartdato(String startdato) {
        this.startdato = startdato;
    }
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/AapenPeriode.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty

@JsonInclude(JsonInclude.Include.NON_NULL)
data class AapenPeriode(
    @JsonProperty("type") var ukjentEllerÅpenSluttdato: String? = null,
    @JsonProperty("startdato") var startdato: String? = null
)
